deltachat-jsonrpc/src/api/mod.rs

```diff
@@ -1750,10 +1750,15 @@ impl CommandApi {
         account_id: u32,
         chat_id: u32,
         sticker_path: String,
+        force: bool,
```

Comment:
This could also be a new function to not break desktop, but I think this adjustment is fine @Simon-Laux

Reply:
yeah as long as it is mentioned as breaking api change in the changelog